src/client/results/mostfrequent/MostFrequent.js

```diff
@@ -1,9 +1,19 @@
 import React, { Component } from 'react';
 import SlidingSearchBar from './SearchBar';
 
-const MostFrequent = ({ title, filterTerm, onTypingInSearchBar, onToggleAll, currentToggledAction, children }) => (
-  <div>
-    <h3>{title}</h3>
+const MostFrequent = ({
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [This function has too many parameters. (8)](https://www.codacy.com/app/winwardo/solid-octo-disco/file/2384361377/issues/source?bid=3148782&fileBranchId=3227421#l4)